IDEAS/Fluid/Taps/TapProfile.mo

```diff
@@ -0,0 +1,84 @@
+within IDEAS.Fluid.Taps;
+model TapProfile
+  "Model for a DHW tap, reading the DHW demand profile via a CombiTimeTable."
+  extends IDEAS.Fluid.Taps.BaseClasses.PartialTap;
+
+  parameter Real uniCon=1/60
+    "Conversion factor to convert units from DHW profile [l/min] to [kg/s]";
+  parameter String loadFile=Modelica.Utilities.Files.loadResource(
+    "modelica://IDEAS/Resources/domestichotwaterprofiles/DHW_1year_2adults.txt") annotation(Dialog(loadSelector(filter="All files (*.*)", caption="Select the DHW profile file")));
+
+  Modelica.Blocks.Sources.CombiTimeTable pro(
```

Comment:
demPro or something like that. `pro` is maybe a bit too vague

Reply:
https://github.com/open-ideas/IDEAS/pull/1355/commits/1c66d8fd124519bbef7c5777e27cbc1c5e3de2e5